package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "material-angular-dashboard",
+  "version": "0.0.0",
+  "description": "Free dashboard template with Material Design Lite and Angular",
+  "keywords": [
+    "dashboard",
+    "admin template",
+    "mdl",
+    "getmdl",
+    "Material Design",
+    "Material Design Lite",
+    "Dark",
+    "dark material design",
+    "angular",
+    "ng"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "lint": "ng lint"
```

Comment:
- recommend adding `"lint:fix": "ng lint --fix"` rule
- add [stylelint](https://github.com/stylelint/stylelint) for scss files

Reply:
✔️